### DISCUSSION TITLE
blog/2024/01/first-blog/

### DISCUSSION THREAD

**Discussion by giscus** [userliluzibert/userliluzibert.github.io#9] [Giscus]
# blog/2024/01/first-blog/

"[Torc waterfall, co. Kerry, Ireland #photo #photography #iphone #iphone5s #iphoneography #ireland #landscape #travel #outdoor #nature #sky #clouds #lake(https://www.flickr.com/photos/87690240@N03/14133418959)]" by Giuseppe Milo (www.pixael.com) is licensed under CC BY 2.0.
I want to be able to visit my website under the /blog subdirectory and find some stuff here. So I’m going to try to separate things; /docs for documentation and guides; leaving /blog for everything else, mostly me typing out what pops into my head and whatever I find interesting.

https://blog.liluzibird.dev/blog/2024/01/first-blog/

**Comment by armando013005** [comment]
jknkjhgcfxdzsAsdafghjkl;'
.?nbcvfxgdserw54e6

**Reply by liluzibird** [reply]
Thank you for your input. I agree completely.
